dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -189,6 +190,7 @@ public void testWorkflowMigration() throws Exception {
      * This tests the GitHub release process
      */
     @Test
+    @Ignore("Test keeps failing on CI, enable later. https://ucsc-cgl.atlassian.net/browse/SEAB-1873")
```

Comment:
this seems like a pretty long test testing a variety of operations
can it be broken up rather than disabling the whole thing?

Reply:
It's going to take quite a long time to figure out what's wrong with this test.  Considering it's been broken on develop for the past 4 commits, it seems irrelevant to the PR.

Reply:
Agreed, I think it should be fixed in a separate PR.